sanic/app.py

```diff
@@ -111,7 +112,7 @@ def decorator(listener):
 
     # Decorator
     def route(self, uri, methods=frozenset({'GET'}), host=None,
-              strict_slashes=False, stream=False, version=None):
+              strict_slashes=None, stream=False, version=None):
```

Comment:
yeah, it works in your case, because that is inside `def route(self, ...)`, `self` is available then.
```python
def route(self, ...):
   # blablabla
   self.router.add(..., merge_routes=self.merge_routes)
```
Correct me if i am looking at wrong place.